### PR TITLE
Website for version 0.3.0

### PR DESCRIPTION
**Pull request by thefosk**
Documentation for the upcoming release `0.3.0`.

### CODE REVIEW

File: app/plugins/http-log.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+sitemap: true
+id: page-plugin
+title: Plugins - HTTP Log
+header_title: HTTP Log
+header_icon: /assets/images/icons/plugins/http-log.png
+header_btn_text: Report Bug
```
Comment (ahmadnassri) [inline app/plugins/http-log.md]: when #84 gets merged, you can remove the following 3 lines

File: app/plugins/http-log.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (ahmadnassri) [inline app/plugins/http-log.md]: when #84 gets merged, you can remove the following lines

```
sitemap: true
id: page-plugin
```

```
header_btn_text: Report Bug
header_btn_href: https://github.com/Mashape/kong/issues/new
header_btn_target: _blank
breadcrumbs:
  Plugins: /plugins
```

File: app/plugins/ssl.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+sitemap: true
```
Comment (ahmadnassri) [inline app/plugins/ssl.md]: when #84 gets merged, you can remove the following lines

```
sitemap: true
id: page-plugin
```

```
header_btn_text: Report Bug
header_btn_href: https://github.com/Mashape/kong/issues/new
header_btn_target: _blank
breadcrumbs:
  Plugins: /plugins
```

Comment (ahmadnassri) [inline app/plugins/ssl.md]: #84 is merged, @rainum can you please rebase and adjust this accordingly?

**Comment by thibaultCha** [comment]
@shashiranjan84 @thefosk When documenting plugins we should stop using the terrible `/plugins_configurations/` route still with than dammed `api_id`.

Use `/apis/{api}/plugins/`

**Pull request merged**